nomenclature/cli.py

```diff
@@ -29,4 +31,4 @@ def cli_valid_project(path: Path):
        in a model mapping are defined in the region codelist.
```

Comment:
Adding an example here for non-standard use that will be rendered on https://nomenclature-iamc.readthedocs.io/en/stable/cli.html would be great.
Something like
```python
"""
...
    Example
    -----------
    $ nomenclature validate-project . --mappings mapping --definitions definitions

"""
```